mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -714,6 +718,13 @@ The `node_config` block supports:
   Structure is documented below.
   Cannot be updated.
 
+* `enable_ip_masq_agent` -
+  (Optional)
+  Deploys 'ip-masq-agent' daemon set in the GKE cluster and defines
```

Comment:
This is not true is Composer 2 (while it is in Composer 1).

Maybe something like:
IP Masq Agent translates Pod IP addresses to node IP addresses, so that destinations and services targeted from Airflow DAGs and tasks only receive packets from node IP addresses instead of Pod IP addresses